dnf-behave-tests/fixtures/specs/dnf-ci-fedora/man-pages-4.16-3.fc29.spec

```diff
@@ -0,0 +1,23 @@
+Name: man-pages
```

Comment:
Please don't add packages to `dnf-ci-fedora`. We were just discussing this - the repository is used for a large number of tests, which get influenced if you change its data (however improbably it is that this will silently break something, the chance is there). The solution should be to create your own repository (which we've agreed should be named e.g. `microdnf-install-nodocs` for this test, and the feature files could then be named like `microdnf/install-nodocs-1.feature` and `microdnf/install-nodocs-2.feature` in this case).